tests/calculations/test_singlepoint.py

```diff
@@ -0,0 +1,138 @@
+"""Tests for singlepoint calculation."""
+
+import subprocess
+
+from ase.build import bulk
+import pytest
+
+from aiida.common import datastructures
+from aiida.engine import run
+from aiida.orm import Str, StructureData
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.data.model import ModelData
+
+
+def test_singlepoint(fixture_sandbox, generate_calc_job, tmp_path, janus_code):
+    """Test generating singlepoint calculation job"""
+    # pylint:disable=line-too-long
+    entry_point_name = "janus.sp"
+    inputs = {
+        "metadata": {"options": {"resources": {"num_machines": 1}}},
+        "code": janus_code,
+        "architecture": Str("mace_mp"),
+        "precision": Str("float64"),
+        "structure": StructureData(ase=bulk("NaCl", "rocksalt", 5.63)),
+        "calctype": Str("singlepoint"),
+        "model": ModelData.download(
+            "https://github.com/stfc/janus-core/raw/main/tests/models/mace_mp_small.model",
+            architecture="mace_mp",
+            cache_dir=tmp_path,
```

Comment:
I haven't looked into it yet, but if you replace `tmp_path` with `str(tmp_path)` this fails (`TypeError: expected str, bytes or os.PathLike object, not NoneType`) which suggests that the str version doesn't work.

Reply:
Doesn't seem to be a problem actually, works either way now.